NumWords.java

```diff
@@ -2,5 +2,13 @@
 public class NumWords {
 	public static void main(String args[]) {
 	    // Replace this comment with your code
+		int number1 = Integer.parseInt(args[0]);
+		int o = number1 % 10;
+		int t = ((number1 % 100)- o)/10 ;
+		int h = ((number1%1000)-o -t)/100;
```

Comment:
Spaces between operations!
Please read the Java styling guidelines document in the course's Moodle page